### DISCUSSION TITLE
HyperHDR 20beta1 not available for armv7 CPUs

### DISCUSSION THREAD

**Discussion by rama31244** [awawa-dev/HyperHDR#694] [Q&A]
HI
I run raspberry pi OS bullseye with an ARMv7 chip but there doesn't appear to be an ARMv7 version of this release yet. Can i use the ARMv6 version?

**Comment by awawa-dev** [comment]
Hi
Raspberry Pi OS comes only in two major version: arm6 (let's say 32bits) and aarch64/arm64. Assuming that you want to use arm7, you are using a 32-bit OS, so arm6 should work for you although the 64-bit OS system and the aarch64 version of HyperHDR offer higher performance.

**Reply by rama31244** [reply]
I ended up using the Linux repo to install it so I'm guessing it just installed the arm6 version since it appears to be working. Thanks

**Reply by awawa-dev** [reply]
Indeed, installing/updating using the repository is the safest&fastest way. Also the APT repository is the main reason for removing the armv7 version as it does not match any official APT architecture.
